Travis fails with IPython 7.11

<!--- **PLEASE READ:** When submitting here, please ensure you've completed the following checklist and checked the boxes to confirm. Issue reports without it may be closed. Thanks! --->

### Issue Report Checklist

* [ ] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [ ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [ ] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [ ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below


## Problem Description
In cython tests, I get the message:
```
File "/home/travis/miniconda/envs/test/lib/python3.7/site-packages/Cython/Build/IpythonMagic.py", line 409, in _cythonize

E               pyx_file = py3compat.cast_bytes_py2(pyx_file, encoding=sys.getfilesystemencoding())

E           

E           AttributeError: module 'IPython.utils.py3compat' has no attribute 'cast_bytes_py2'
```
(see https://travis-ci.org/spyder-ide/spyder/jobs/630165407?utm_medium=notification&utm_source=github_status)
### What steps reproduce the problem?

1. 
2. 
3. 

### What is the expected output? What do you see instead?



### Paste Traceback/Error Below (if applicable)
<!--- Copy from error dialog or View > Panes > Internal Console --->

```python-traceback

PASTE TRACEBACK HERE

```

## Versions
<!--- You can get this information from Help > About Spyder...
or (if Spyder won't launch) the "conda list" command
from the Anaconda Prompt/Terminal/command line. --->

* Spyder version: 
* Python version: 
* Qt version: 
* PyQt version: 
* Operating System name/version: 

### Dependencies
<!--- Please go to the menu entry Help > Dependencies,
press the Copy to clipboard button and paste below --->

```
PASTE DEPENDENCIES HERE
```

--- a/spyder/app/tests/test_mainwindow.py
+++ b/spyder/app/tests/test_mainwindow.py
@@ -11,6 +11,7 @@ Tests for the main window.
 """
 
 # Standard library imports
+from distutils.version import LooseVersion
 import os
 import os.path as osp
 import re
@@ -25,6 +26,7 @@ import sys
 import uuid
 
 # Third party imports
+from IPython.core import release as ipy_release
 from flaky import flaky
 from jupyter_client.manager import KernelManager
 import numpy as np
@@ -835,9 +837,10 @@ def test_change_cwd_explorer(main_window, qtbot, tmpdir, test_directory):
 
 @pytest.mark.slow
 @flaky(max_runs=3)
-@pytest.mark.skipif((os.name == 'nt' or not is_module_installed('Cython') or
-                     sys.platform == 'darwin'),
-                    reason="Hard to test on Windows and macOS and Cython is needed")
+@pytest.mark.skipif(
+    (os.name == 'nt' or sys.platform == 'darwin' or
+     LooseVersion(ipy_release.version) == LooseVersion('7.11.0')),
+    reason="Hard to test on Windows and macOS and fails for IPython 7.11.0")
 def test_run_cython_code(main_window, qtbot):
     """Test all the different ways we have to run Cython code"""
     # ---- Setup ----

--- a/spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py
+++ b/spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py
@@ -12,6 +12,7 @@ Tests for the IPython console plugin.
 
 # Standard library imports
 import codecs
+from distutils.version import LooseVersion
 import os
 import os.path as osp
 import shutil
@@ -341,7 +342,10 @@ def test_sympy_client(ipyconsole, qtbot):
 
 @flaky(max_runs=3)
 @pytest.mark.cython_client
-@pytest.mark.skipif(os.name == 'nt', reason="It doesn't work on Windows")
+@pytest.mark.skipif(
+    (os.name == 'nt' or
+     LooseVersion(ipy_release.version) == LooseVersion('7.11.0')),
+    reason="Doesn't work on Windows and fails for IPython 7.11.0")
 def test_cython_client(ipyconsole, qtbot):
     """Test that the Cython console is working correctly."""
     # Wait until the window is fully up
@@ -1169,6 +1173,7 @@ def test_sys_argv_clear(ipyconsole, qtbot):
 
 
 @flaky(max_runs=5)
+@pytest.mark.skipif(os.name == 'nt', reason="Fails on Windows")
 def test_set_elapsed_time(ipyconsole, qtbot):
     """Test that the IPython console elapsed timer is set correctly."""
     shell = ipyconsole.get_current_shellwidget()

